src/jit/utils.cpp

```diff
@@ -1954,6 +1954,68 @@ float FloatingPointUtils::round(float x)
     return _copysignf(flrTempVal, x);
 }
 
+bool FloatingPointUtils::isNormal(double x)
+{
+    int64_t bits = *(int64_t*)&x;
```

Comment:
`reinterpret_cast` should be used instead of C style cast

Reply:
done. replaced with `reinterpret_cast<uint64_t &>(x)`